src/index.ts

```diff
@@ -98,7 +102,14 @@ export function throttling(octokit: Octokit, octokitOptions = {}) {
   const events = {};
   const emitter = new Bottleneck.Events(events);
   // @ts-ignore
-  events.on("abuse-limit", state.onAbuseLimit);
+  events.on(
+    "secondary-limit",
```

Comment:
I'm assuming `secondary-limit` event will be sent from the  L145 (so we have full control of the name). Is that correct or it should be listening to both (["abuse-limit", "secondary-limit"]?